plugins/woocommerce/includes/admin/meta-boxes/views/html-product-data-panel.php

```diff
@@ -37,21 +38,39 @@
 			</label>
 		<?php endforeach; ?>
 	</span>
+	
+	<?php if ( true ) : ?>
+		<dl class="product_data_cards wc-cards">
+			<?php foreach ( self::get_product_data_tabs() as $key => $tab_data ) : ?>
+				<dt class="clear <?php echo esc_attr( $key ); ?>_options_label <?php echo esc_attr( $key ); ?>_card_label <?php echo esc_attr( isset( $tab_data['class'] ) ? implode( ' ', (array) $tab_data['class'] ) : '' ); ?>">
+					<span><?php echo esc_html( $tab_data['label'] ); ?></span>
+				</dt>
+				<dd class="card <?php echo esc_attr( $key ); ?>_options <?php echo esc_attr( $key ); ?>_card <?php echo esc_attr( isset( $tab_data['class'] ) ? implode( ' ', (array) $tab_data['class'] ) : '' ); ?>">
+					<?php self::output_tab( $tab_data ); ?>
+				</dd>
+			<?php endforeach; ?>
+		</dl>
+		<script type="text/javascript">
+			for (let hiddenCard of document.querySelectorAll( '.product_data_cards .card > .hidden' ) ) {
+				hiddenCard.classList.remove( 'hidden' );
```

Comment:
This hacky inline script tag is just so I don't have to touch each of the tab files to remove 'hidden' and then have to satisfy the linter on each of those files.